src/cpp/tests/automation/testthat/test-automation-completions.R

```diff
@@ -28,3 +28,26 @@ test_that("autocompletion doesn't trigger active bindings", {
    expect_false(tail(output, 1) == "[1] \"active\"")
    
 })
+
+# https://github.com/rstudio/rstudio/issues/13611
+test_that("autocompletions within piped expressions work at start of document", {
+   
+   code <- .rs.heredoc('
+      
+      mtcars |> mutate(x = mean())
+   ')
+   
+   remote$documentOpen(".R", code)
+   editor <- remote$editorGetInstance()
+   editor$gotoLine(2L, 26L)
+   remote$keyboardExecute("<Tab>")
+   popupEl <- remote$jsObjectViaSelector("#rstudio_popup_completions")
+   popupText <- popupEl$innerText
+   expect_true(grepl("trim =",  popupText, fixed = TRUE))
+   expect_true(grepl("na.rm =", popupText, fixed = TRUE))
+   expect_true(grepl("... =",   popupText, fixed = TRUE))
```

Comment:
```suggestion
   code <- .rs.heredoc('
      
      mtcars |> mutate(x = mean())
      mtcars |> mutate(x = mean())
   ')
   
   remote$documentOpen(".R", code)
   editor <- remote$editorGetInstance()
   editor$gotoLine(2L, 26L)
   remote$keyboardExecute("<Tab>")
   popupEl <- remote$jsObjectViaSelector("#rstudio_popup_completions")
   popupText <- popupEl$innerText
   expect_true(grepl("x =",  popupText, fixed = TRUE))
   expect_true(grepl("... =",   popupText, fixed = TRUE))

   editor$gotoLine(3L, 26L)
   remote$keyboardExecute("<Tab>")
   popupEl <- remote$jsObjectViaSelector("#rstudio_popup_completions")
   popupText <- popupEl$innerText
   expect_true(grepl("x =",  popupText, fixed = TRUE))
   expect_true(grepl("... =",   popupText, fixed = TRUE))
```

The test should possibly look like this